scripts/bridge-setup/bridge-setup.go

```diff
@@ -29,58 +29,67 @@ func main() {
 	privateKey := flag.String("privateKey", "0x0", "Hedera Private Key")
```

Comment:
I think that it would be better to have a `scripts/bridge` folder and place the `setup.go` file there. This would be a more go idiomatic way of doing it